.github/workflows/reusable-license-checker.yml

```diff
@@ -0,0 +1,33 @@
+# Copyright The Notary Project Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Reusable license checker
+
+on:
+  workflow_call:
+
+jobs:
+  check-license:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Check license header
+        uses: apache/skywalking-eyes/header@v0.4.0
```

Comment:
Please do not use `skywalking-eyes/header@v0.4.0`, I've detected couple of bugs of this version. 
Instead of writing another `reusable-license-checker.yml`, we should reuse this one: https://github.com/notaryproject/notation-core-go/blob/main/.github/workflows/reusable-license-checker.yml.
An example of how to use it: https://github.com/notaryproject/notation-go/blob/main/.github/workflows/license-checker.yml